crt_portal/cts_forms/templates/forms/complaint_view/show/activity_stream.html

```diff
@@ -1,5 +1,13 @@
 {% extends "forms/complaint_view/show/card.html" %}
 {% block title %}{{title}}{% endblock %}
+{% block extra_title %}
+  <div class="card-controls">
+    <span class="usa-tooltip" data-position="right" title="Include system messages in the activity log">
+      <input checked="checked" type="checkbox" name="show_system" class="usa-checkbox__input" aria-label="Include system messages in activity" id="id_activity_system">
+      <label for="id_activity_system" class="usa-checkbox__label crt-checkbox__label">System</label>
```

Comment:
Maybe this could say "Comments only", default to unchecked and filter out the other actions when checked to only see comments?

Reply:
Updated! ![activity](https://github.com/usdoj-crt/crt-portal-management/assets/15126660/f150d095-03e2-4800-aa9d-c2a329cf1310)